packages/platform/platform-fastify/src/services/PlatformFastifyRequest.ts

```
@@ -0,0 +1,58 @@
+import {PlatformContext, PlatformRequest} from "@tsed/common";
+import {FastifyRequest} from "fastify";
+import {IncomingMessage} from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)